src/mozilla_sec_eia/models/sec10k/utils/cloud.py

```diff
@@ -62,6 +62,12 @@ def from_10k(cls, filename: str, sec10k_text: str, ex_21_version: str):
 
     def save_as_pdf(self, file: BinaryIO):
         """Save Exhibit 21 as a PDF in `file`, which can be in memory or on disk."""
+        # TODO: probably should make a "corrections" file that has CSS/HTML replacements
+        # to make PDF render
+        if "border-bottom: black thin solid;" in self.ex_21_text:
+            self.ex_21_text = self.ex_21_text.replace(
+                "border-bottom: black thin solid;", "border-bottom: 1px solid black;"
+            )
```

Comment:
this is a super janky fix to make one of the Ex.21 PDFs render. I imagine that we will probably want to have a separate cleaning function with a dictionary of common CSS/HTML replacements that make the Ex. 21 PDF render.

Reply:
Yeah there's a bunch of cases in the full extraction where things didn't render properly. I might be able to add some better logging to capture these better in the future.